src/main/java/ru/hse/sd/cli/Parser.java

```diff
@@ -0,0 +1,164 @@
+package ru.hse.sd.cli;
+
+import ru.hse.sd.cli.exceptions.ParserException;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+/**
+ * Class that parse String to the RawArgs (this is arguments that are not
+ * prepared yet, that means there wasn't any substitution or equating)
+ */
+public class Parser {
+    private final char DOUBLE_QUOTES = '\"';
+    private final char SINGLE_QUOTES = '\'';
+
+
+    private int cntOfSymbols(String line, char symbol) {
```

Comment:
```suggestion
    private final char SINGLE_QUOTES = '\'';

    private int cntOfSymbols(String line, char symbol) {
```
Две пустые строки подряд обычно не ставят